bitnami/harbor/Chart.yaml

```diff
@@ -34,4 +34,4 @@ sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/harbor-registry
   - https://github.com/bitnami/containers/tree/main/bitnami/harbor-registryctl
   - https://goharbor.io/
-version: 15.0.4
+version: 15.0.5
```

Comment:
Since this is adding a new feature, could you increase the minor version instead? I.e. to `15.1.0`.

Reply:
Done